sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -25,6 +26,11 @@ public Discount getDiscount(@PathVariable Long id) {
         return service.getAll();
     }
 
+    @GetMapping("/search")
+    public List<Discount> search(@RequestParam String searchText) {
+        return service.getByNameOrDescriptionContaining(searchText);
```

Comment:
Let's use more abstract name `search` or `find`. Logic of this method can be improved and it doesn't mean that we should change the name each time